### PR TITLE
Remove tournaments and related components

### LINKED ISSUE #3: Clone master to different branch and remove tournaments
The focus has shifted from tournaments to a ranking system. As part of the restructuring that comes with this we need to remove code that's not related/necessary for this.
We need to do the following:

- [x] Clone master to a different branch
- [x] Remove tournaments and teams from working branches that are still to be merged

### PR DESCRIPTION
**Pull request by czarnecki**
This removes tournaments and all components
Closes #3

### CODE REVIEW

File: src/main/java/de/adesso/kicker/notification/Notification.java [hunk 1]
```diff
@@ -1,88 +0,0 @@
...
-        this.type = type;
-    }
-
-    @Override
-    public String toString() {
-        return "Notification{" + "notificationId=" + notificationId + ", notificationType=" + ", receiver=" + receiver
-                + ", sender=" + sender + ", sendDate=" + sendDate;
-
-    }
-}
```
Comment (Flo4l) [inline src/main/java/de/adesso/kicker/notification/Notification.java]: Notifications should not be removed completely

File: src/main/java/de/adesso/kicker/team/exception/IdenticalPlayersException.java [hunk 2]
```diff
@@ -1,9 +0,0 @@
-package de.adesso.kicker.team.exception;
-
-public class IdenticalPlayersException extends RuntimeException {
-
-    public IdenticalPlayersException() {
-        super("You cannot play with identical players.");
-    }
-
-}
```
Comment (Flo4l) [inline src/main/java/de/adesso/kicker/team/exception/IdenticalPlayersException.java]: Maybe still relevant for Matches?

Comment (czarnecki) [reply src/main/java/de/adesso/kicker/team/exception/IdenticalPlayersException.java]: Maybe but it doesn't work currently so it's better to remove it for now and reimplement it later.

File: src/main/java/de/adesso/kicker/notification/Notification.java (hunk 1 above)
Comment (czarnecki) [reply src/main/java/de/adesso/kicker/notification/Notification.java]: Even if that's the case I think it's still better to remove it and see if there's a better way to do them.

**Pull request merged**